PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -0,0 +1,1111 @@
+    hCollisions = registry.add<TH1>("hCollisions", "HF event counter;;entries", {HistType::kTH1D, {axisEvents}});
+    hPosZBeforeEvSel = registry.add<TH1>("hPosZBeforeEvSel", "all events;#it{z}_{prim. vtx.} (cm);entries", {HistType::kTH1D, {{400, -20., 20.}}});
+    hPosZAfterEvSel = registry.add<TH1>("hPosZAfterEvSel", "selected events;#it{z}_{prim. vtx.} (cm);entries", {HistType::kTH1D, {{400, -20., 20.}}});
+    hPosXAfterEvSel = registry.add<TH1>("hPosXAfterEvSel", "selected events;#it{x}_{prim. vtx.} (cm);entries", {HistType::kTH1D, {{200, -0.5, 0.5}}});
+    hPosYAfterEvSel = registry.add<TH1>("hPosYAfterEvSel", "selected events;#it{y}_{prim. vtx.} (cm);entries", {HistType::kTH1D, {{200, -0.5, 0.5}}});
+    hNumPvContributorsAfterSel = registry.add<TH1>("hNumPvContributorsAfterSel", "selected events;#it{y}_{prim. vtx.} (cm);entries", {HistType::kTH1D, {{500, -0.5, 499.5}}});
+
+    /// collision monitoring
+    setLabelHistoEvSel(hCollisions);
+
+    df.setPropagateToPCA(propagateToPCA);
+    df.setMaxR(maxR);
+    df.setMaxDZIni(maxDZIni);
+    df.setMaxDXYIni(maxDXYIni);
+    df.setMinParamChange(minParamChange);
+    df.setMinRelChi2Change(minRelChi2Change);
+    df.setMaxChi2(maxChi2);
+    df.setUseAbsDCA(useAbsDCA);
+    df.setWeightedFinalPCA(useWeightedFinalPCA);
+    df.setRefitWithMatCorr(refitWithMatCorr);
+
+    ccdb->setURL(ccdbUrl);
+    ccdb->setCaching(true);
+    ccdb->setLocalObjectValidityChecking();
+    lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbPathLut));
+    runNumber = 0;
+  }
+
+  template <o2::aod::hf_collision_centrality::CentralityEstimator centEstimator, int decayChannel, typename Coll, typename Hist>
+  void runXic0Omegac0Creator(Coll const&,
+                             aod::BCsWithTimestamps const&,
+                             TracksWCovDca const&,
+                             MyCascTable const&, CascadesLinked const&,
+                             aod::HfCascLf2Prongs const& candidates,
+                             Hist& hInvMassCharmBaryon,
+                             Hist& hFitterStatus,
+                             Hist& hCandidateCounter,
+                             Hist& hCascadesCounter)
+  {
+
+    if constexpr (decayChannel != hf_cand_casc_lf::DecayType2Prong::XiczeroOmegaczeroToXiPi && decayChannel != hf_cand_casc_lf::DecayType2Prong::OmegaczeroToOmegaPi && decayChannel != hf_cand_casc_lf::DecayType2Prong::OmegaczeroToOmegaK) {
+      LOGP(fatal, "Decay channel not recognized!");
+    }
+
+    double massPionFromPDG = MassPiPlus;      // pdg code 211
+    double massKaonFromPDG = MassKPlus;       // pdg code 321
+    double massLambdaFromPDG = MassLambda0;   // pdg code 3122
+    double massXiFromPDG = MassXiMinus;       // pdg code 3312
+    double massOmegaFromPDG = MassOmegaMinus; // pdg code 3334
+    double massOmegacFromPDG = MassOmegaC0;   // pdg code 4332
+    double massXicFromPDG = MassXiC0;         // pdg code 4132
```

Comment:
There is no need to redefine these variables, you can directly use those on the right of the `=`

Reply:
Okay, change implemented